addins/Cake.Scripting.Abstractions.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Scripting.Abstractions
```

Comment:
Should be filtered as it is part of Cake and not an addin

Reply:
Done. Cake.Scripting.Abstractions added to blacklist